src/axom/primal/operators/winding_number.hpp

```diff
@@ -7,7 +7,7 @@
  * \file winding_number.hpp
  *
  * \brief Consists of methods to compute the generalized winding number (GWN) 
- *        for points with respect to various geometric objects.
+ *        for points with respect to various 2D geometric objects.
```

Comment:
Clarification: This file has both 2D and 3D functions, right?

Reply:
Whoops! Fixed that.